### DISCUSSION TITLE
How to install a github bot?

### DISCUSSION THREAD

**Discussion by Rosey723** [community/community#130928] [Programming Help]
### Body

Haven;t had much luck installing bots from github so i want to do this without any chance of mistake. How do i install this, step-by-step? I'd like to install it using cmd if that's possible? 
https://github.com/edmundj0/resy-reservations-bot

Clone this repository ( already downloaded as a zip )

Install dependencies ( How do i install dependencies using cmd or another software )
pipenv install

Create .env file in root directory, and create EMAIL and PASSWORD variables based on your resy log in info ( how do i creat an .env file. do i just type in my email and password in this file )

EMAIL=example@example.com
PASSWORD=examplePassword000

Insert values into config.ini file

Venue: 0000
Date: 2023-12-31
Guests: 2
Note: Venue Id can be found in the network tab from the restaurant page on resy. Date is in YYYY-MM-DD format

Enter virtual environment and run script, or run script directly ( can i use cmd for this, or do i have to download a new software? )

pipenv shell
python3 resy_bot.py
OR

pipenv run python3 resy_bot.py
Failed attempts will be logged in attempts.csv . Leave script running.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by zhao-stanley** [comment]
Create a new file and name it `.env`, no extension. And then instead of example@example.com and examplePassword000, you'd fill in your own credentials. Be careful not to commit your `.env` file to Github.

**Reply by Rosey723** [reply]
how do i install dependencies and how do i run a script directly? using cmd?
